vocode/streaming/action/factory.py

```diff
@@ -1,11 +1,14 @@
 from vocode.streaming.action.base_action import BaseAction
 from vocode.streaming.action.nylas_send_email import NylasSendEmail
+from vocode.streaming.action.transfer_call import TransferCall
 from vocode.streaming.models.actions import ActionType
 
 
 class ActionFactory:
     def create_action(self, action_type: str) -> BaseAction:
         if action_type == ActionType.NYLAS_SEND_EMAIL:
             return NylasSendEmail(should_respond=True)
+        elif action_type == ActionType.TRANSFER_CALL:
+            return TransferCall(to_phone="+15555555555")
```

Comment:
we should change this to support an arbitrary transfer call number

Reply:
i think the correct move here is to add an actionconfig (with hard coded params). as a workaround to get the pr in for now, what do you think about including this in the TransferCallParameters?